docs/content/product-vocabulary.stories.mdx

```diff
@@ -240,6 +240,19 @@ When referencing the apps in the App Marketplace, use **connect** and
 | ----------------- | ----------------- |
 | Connect Mailchimp | Turn on Mailchimp |
 
+When referencing user management, use **ractivate** and
```

Comment:
```suggestion
When referencing user management, use **reactivate** and
```

Reply:
I'm actually curious about the automations/campaigns example. "Turn on / Turn off" feels much more "human readable" in those contexts, do you know why we're leaning towards "Activate / Deactivate" in those cases?

Reply:
> I'm actually curious about the automations/campaigns example. "Turn on / Turn off" feels much more "human readable" in those contexts, do you know why we're leaning towards "Activate / Deactivate" in those cases?

The statuses we are using for automated campaigns are "Active", "Inactive", so we went with "activate" / "deactivate" to match.